---
obstacle_msgs/CircleObstacles obstacles